WhatShouldITestTests/MenuUtilityTests.cs

```diff
@@ -0,0 +1,17 @@
+﻿using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using WhatShouldITest.Utilities;
+
+namespace WhatShouldITestTests
+{
+    [TestClass]
+    public class MenuUtilityTests
+    {
+        [TestMethod]
+        public void GetDailySpecial_ItIsMonday_ShouldReturnHalfPriceTacos()
+        {
+            MenuUtilities.GetDailySpecial(DayOfWeek.Monday).Should().Be("Half price tacos!");
```

Comment:
And here we are testing it. At least, we're testing one of the possible paths. An interesting aside; code coverage is now telling us that this function is 100% covered, but in actuality, we're only testing one of the possible paths through this method. There are seven - one for each day of the week. There is also a scenario we should consider where the DayOfWeek parameter is passed in as null.
![image](https://user-images.githubusercontent.com/668946/70745245-02e8f280-1ce9-11ea-8eee-2edb4127df4e.png)

Maybe code coverage isn't the best measurement...